docs/tutorials/quick_install.md

```diff
@@ -187,15 +148,15 @@ During this task, you will prepare the **config.yml** file to configure the envi
 
 1. Run the Terraform plan script to validate changes to be executed.
```

Comment:
if we use bicep then this would be useless

Reply:
I updated the command to ./build.sh -a plan -l bicep